client/src/page/MenuPage.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import styled from "@emotion/styled";
+import MenuImageSrc from "./../assets/asteronautRakete.png";
+import MenuBackgroundImageSrc from "./../assets/planet.png";
+import { Link } from "react-router-dom";
+
+const MenuWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  width: 100vw;
+  align-items: center;
+  justify-content: space-between;
+`;
+
+const MenuImage = styled.img`
+  height: 45vh;
```

Comment:
Try working with rem or %. VH can be tricky

Reply:
Completely agree!